repository/s.json

```diff
@@ -1248,6 +1248,20 @@
 				}
 			]
 		},
+		{
+			"name": "Sf4Helpers",
+			"description": "Symfony 4 command line helpers for Sublime",
```

Comment:
You can remove `description` and it will pull the description from your GitHub repo